jobs/ccg-plugin/templates/pre-start.ps1.erb

```diff
@@ -5,6 +5,12 @@ $ErrorActionPreference = "Stop";
 trap { $host.SetShouldExit(1) }
 
 Write-Output "Started ccg-plugin pre-start" | timestamp
+
+if ([Microsoft.Win32.Registry]::LocalMachine.OpenSubKey("SYSTEM\CurrentControlSet\Control\CCG\COMClasses") -eq $null ) {
```

Comment:
@geofffranks I am curious, did we run into this issue in the pipeline? I don't know how one could run into this unless the stemcell is too old.

Reply:
@winkingturtle-vmw someone ran into it with custom stemcell...